docs/content/styles/spacing.mdx

```diff
@@ -8,7 +8,7 @@ import Spacing from './components/Spacing';
 
 # Spacing
 
-You can refer to a spacing from the theme using `theme.spacings.<COLOR_NAME>`.
+You can refer to a spacing from the theme using `theme.spacings.<size>`.
```

Comment:
Thanks for the fix! We're using uppercase letters [elsewhere](https://circuit.sumup.com/#/styles/colors) to highlight that the user has to fill in the desired size/color/etc. Could you please do the same here? :)

```suggestion
You can refer to a spacing from the theme using `theme.spacings.<SIZE>`.
```